_data/cryptocurrencies.yml

```
@@ -568,6 +568,15 @@ websites:
     regions:
       - ng
 
+  - name: NDAX
+    url: https://ndax.io/
+    img: ndax.svg
+    tfa:
+      - totp
+    doc: https://help.ndax.io/en-ca/article/setting-up-google-authenticator-for-2fa-mfz047/
```

```suggestion
    doc: https://help.ndax.io/en-ca/category/2fa-rgk6to/
```
I think that this page may be slightly more useful